Tripal Governance including the Steering Committee

laceysanderson opened the pull request:
# Documentation 

Issue #786 

### LEAVE OPEN UNTIL March 19, 2019

## Description
This PR adds the current version of the Tripal Governance structure to the official documentation. The interm steering committee has developed this documentation based on feedback from the community including discussion on #786 and #344 and during multiple Tripal User Meetings.

We encourage further feedback through this PR. If you have any concerns or would like further explanation, please comment promptly on this PR -we would Love to hear from you. If there has been no objection by March 19 and this document has been approved by the Tripal Project Management Committee (TPMC) then it will be merged and become binding to the community.

## Testing?
Please read through the [Tripal Governance Structure](https://tripal.readthedocs.io/en/786-governance/dev_guide/contributing/governance.html) and [Proposal writing guidelines](https://tripal.readthedocs.io/en/786-governance/dev_guide/contributing/funding.html). I rendered this PR using ReadtheDocs to make review easier. You can also compare these pages to the agreed upon [GoogleDoc](https://docs.google.com/document/d/1iRGmZzKnAm8c5Q9yYLiTctSJaRqnT6Bm0eix1uzoTxw/edit?usp=sharing)

## Screenshots (if appropriate):
![Screen Shot 2019-03-12 at 6 51 33 PM](https://user-images.githubusercontent.com/1566301/54245761-bbce9680-44f7-11e9-8926-7032a01295ad.png)

laceysanderson committed:
Add documentation for Tripal Governance including the Steering Committee.

mestato reviewed (approved):
Looks great! Also, so wonderful to have documentation in the repo as part of the codebase!

mestato commented:
This is version approved by the acting TSC co-chairs. We are excited to hear feedback from the community!

bradfordcondon commented on docs/dev_guide/contributing/governance.rst:
```diff
@@ -1,19 +1,138 @@
-- Committers: write access to the repository.
-- PMC: Makes **code relevant** decisions.
-- Oversight committee: Makes **policy level** decisions.  This may overlap with PMC, but the idea is the oversight committee includes parties who are **not necessarily coders** and therefore not reviewing Pull requests etc.
+These guidelines are binding to the Tripal Community. If you have comments or questions, please `comment on the Github issue <https://github.com/tripal/tripal/issues/786>`_. Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change.
```
I would change this to perhaps be "please create an issue for discussion on github or bring your comments/questions to the user meeting", as #786 and the other governance issues are for the formation of the TAC etc, not to serve as a permanent discussion board for it.


```suggestion
These guidelines are binding to the Tripal Community. If you have comments or questions, please `post them as a Github issue <https://github.com/tripal/tripal/issues/new?template=discussion.md>`_ or ask them at the user's meeting. Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change.
```

bradfordcondon reviewed (approved):
This PR looks good and reflects the consensus reached in the google doc.  Thank you!

bradfordcondon committed:
Update docs/dev_guide/contributing/governance.rst

Co-Authored-By: laceysanderson <las166@mail.usask.ca>

spficklin commented:
Wonderful!  I like it.  I do have some suggested changes

-  Can we move this documentation out of the Developer's Guide?  I would like to see this having greater visibility. Perhaps we should add a new section at the same level of the Developer's Guide and User's Guide called 'Tripal Community'.  One selling point of Tripal is that we are a community and I think if we moved this document out it would help reflect that better.
- I think this text should be reworded at the very end of the document "Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change." Shouldn't changes to the document be made in conjunction with the TSC and the TPMC?
- I don't understand what the word "binding" means when we say the document is binding on the community.  It's not a legal document.  Perhaps it could read instead "These guidelines define the structure for official Tripal Management and Governance"
- Renamed the doc "Proposal development" to "Funding Proposal Development" so it's clear what kind of proposals is meant.

laceysanderson committed:
Make governance docs more prominent.

laceysanderson commented:
Thanks for the suggestions @spficklin!


>     Can we move this documentation out of the Developer's Guide? I would like to see this having greater visibility. Perhaps we should add a new section at the same level of the Developer's Guide and User's Guide called 'Tripal Community'. One selling point of Tripal is that we are a community and I think if we moved this document out it would help reflect that better.

Great idea! Done!

>     I think this text should be reworded at the very end of the document "Changes to this document will be made after adequate discussion has occurred and the project management committee has voted in favor of the change." Shouldn't changes to the document be made in conjunction with the TSC and the TPMC?

I left this one as is as it was discussed during our pre-TSC meeting. Since the TSC is advisory only, it was felt that saying the TPMC was enough and made the process seem less complicated.

>     I don't understand what the word "binding" means when we say the document is binding on the community. It's not a legal document. Perhaps it could read instead "These guidelines define the structure for official Tripal Management and Governance"

Changed 👍 

>     Renamed the doc "Proposal development" to "Funding Proposal Development" so it's clear what kind of proposals is meant.

Changed 👍

spficklin commented:
Merging as we have 3 approvals and even though the Travis CI was not passing it wasn't related to these document changes.
